sources/platform/actors/development/actor_definition/dockerfile.md

```diff
@@ -9,34 +9,48 @@ sidebar_position: 4
+### Node.js base images
 
-- **Node.js 16 on Alpine Linux** ([`apify/actor-node`](https://hub.docker.com/r/apify/actor-node/)) - slim and efficient image, contains only the most elementary tools. Note that headless browsers (Puppeteer, Playwright) are not available in this image.
+Apify provides several Docker images with Node.js, the [Apify SDK for JavaScript](/sdk/js) and [Crawlee](https://crawlee.dev/) preinstalled.
+These images come with either Node.js 16, 18 or 20, you can choose which one you want using one of the `16`, `18` or `20` tags. The `latest` tag corresponds to the latest LTS version of Node.js.
 
-- **Node.js 16 + Puppeteer + Chrome on Debian** ([`apify/actor-node-puppeteer-chrome`](https://hub.docker.com/r/apify/actor-node-puppeteer-chrome/)) - larger image with the Chromium and Google Chrome browsers and the [`puppeteer`](https://github.com/puppeteer/puppeteer) library bundled. With this image, you can use the [`launchPuppeteer()`](https://crawlee.dev/api/puppeteer-crawler/function/launchPuppeteer) function and [`PuppeteerCrawler`](https://crawlee.dev/api/puppeteer-crawler/class/PuppeteerCrawler). Note that Chrome requires quite a lot of resources, therefore the actor should run with at least 2048 MB of memory.
+| Image | Description |
+| ----- | ----------- |
+| Node.js on Alpine Linux ([`actor-node`](https://hub.docker.com/r/apify/actor-node/)) | Slim and efficient image, contains only the most elementary tools. Note that headless browsers (Puppeteer, Playwright) are not available in this image. |
+| Node.js + Puppeteer + Chrome on Debian ([`actor-node-puppeteer-chrome`](https://hub.docker.com/r/apify/actor-node-puppeteer-chrome/)) | Larger image with the Chromium and Google Chrome browsers and the [`puppeteer`](https://github.com/puppeteer/puppeteer) library bundled. |
+| Node.js + Playwright + Chrome on Debian ([`actor-node-playwright-chrome`](https://hub.docker.com/r/apify/actor-node-playwright-chrome/)) | Larger image with the Chromium and Google Chrome browsers and the [`playwright`](https://github.com/microsoft/playwright) library bundled. |
+| Node.js + Playwright + Firefox on Debian ([`actor-node-playwright-firefox`](https://hub.docker.com/r/apify/actor-node-playwright-firefox/)) | Larger image with the Firefox browser and the [`playwright`](https://github.com/microsoft/playwright) library bundled. |
+| Node.js + Playwright + WebKit on Debian ([`actor-node-playwright-webkit`](https://hub.docker.com/r/apify/actor-node-playwright-webkit/)) | Larger image with the Webkit browser engine and the [`playwright`](https://github.com/microsoft/playwright) library bundled. |
+| Node.js + Playwright + all browsers on Debian ([`actor-node-playwright`](https://hub.docker.com/r/apify/actor-node-playwright/)) | A very large and slow image with the [`playwright`](https://github.com/microsoft/playwright) library and all Playwright browsers (Chromium, Chrome, Firefox, WebKit) bundled. |
```

Comment:
```suggestion
| Node.js + Playwright + all browsers on Ubuntu ([`actor-node-playwright`](https://hub.docker.com/r/apify/actor-node-playwright/)) | A very large and slow image with the [`playwright`](https://github.com/microsoft/playwright) library and all Playwright browsers (Chromium, Chrome, Firefox, WebKit) bundled. |
```

Reply:
Ah, cool, didn't know. I'll fix also the Webkit one.